extensions/amp-script/0.1/amp-script.js

```diff
@@ -72,6 +73,21 @@ export const StorageLocation = {
 };
 
 export class AmpScript extends AMP.BaseElement {
+  /** @override @nocollapse */
+  static V1() {
+    return true;
+  }
+
+  /** @override @nocollapse */
+  static deferredBuild() {
+    return false;
```

Comment:
Does it have to be always `!deferredMount`? Above I suggested that we could reconcile `nodom` and "dom" cases.